pkg/scheduling/requirements.go

```diff
@@ -220,15 +220,30 @@ func editDistance(s, t string) int {
 }
 
 func labelHint(r Requirements, key string, allowedUndefined sets.Set[string]) string {
+	var suffix string
+
+	if strings.Contains(key, "/") {
```

Comment:
optional: Make this a function `getSuffix()`?

Reply:
sure